java/api/src/main/java/io/ray/api/Checkpointable.java

```diff
@@ -85,16 +85,4 @@ public Checkpoint(UniqueId checkpointId, long timestamp) {
    *     restart from the beginning.
    */
   UniqueId loadCheckpoint(ActorId actorId, List<Checkpoint> availableCheckpoints);
```

Comment:
@chaokunyang this whole class can be removed.

Reply:
removed